apps/parachain-tracker/src/index.html

```diff
@@ -7,6 +7,7 @@
 
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <link rel="icon" type="image/x-icon" href="favicon.ico" />
+        <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
```

Comment:
This will make the page very slow, we can just use system fonts until we have branding.